presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -146,6 +152,8 @@ Operations
 
     This function doesn't support geometry collections.
 
+    See also:  :func:`geometry_union`, :func:`geometry_union_agg`
```

Comment:
Not a sphinx expert, but `see also` are separate sections in other doc/rst files.